USAGE.md

```diff
@@ -1,39 +1,124 @@
 # How to use avTranscoder
 
-Check out applications contained in the project to see examples of how to use the library in C++, Java or Python.
+## When to use avTranscoder
```

Comment:
Maybe we could write "avTranscoder" in strong characters in the whole document, don't you think ?